requirements-dev.txt

```diff
@@ -18,5 +18,6 @@ pygments==2.1.3
 mock==2.0.0
 requests==2.18.4
 hypothesis==3.56.3
+pluggy<0.7
```

Comment:
I would add a comment about why this dependency is being added here, since it's not really a direct dependency and it's likely in N months we'll forget why it's here.

Reply:
Will do.